documentation/faq/10-what-can-i-make-with-sveltekit.md

```diff
@@ -0,0 +1,22 @@
+---
+title: What can I make with SvelteKit?
+---
+
+SvelteKit can be used to create most kinds of applications. Out of the box, SvelteKit supports many features including:
+
+- Dynamic page content with [load](/docs/load) functions and [API routes](/docs/routing#server).
+- SEO-friendly dynamic content with [server-side rendering (SSR)](/docs/glossary#ssr).
+- User-friendly progressively-enhanced interactive pages with SSR and [Form Actions](/docs/form-actions).
+- Static pages with [prerendering](/docs/page-options#prerender).
+
+SvelteKit can also be deployed to a wide spectrum of hosted architectures via [adapters](/docs/adapters). In cases where SSR is used or server-side logic is added without using prerendering, those functions will be adapted to the target backend. Some examples include:
```

Comment:
I really struggled to read this sentence for some reason, @FractalHQ suggested adding brackets

```suggestion
SvelteKit can also be deployed to a wide spectrum of hosted architectures via [adapters](/docs/adapters). In cases where SSR is used (or server-side logic is added without prerendering), those functions will be adapted to the target backend. Some examples include:
```

Reply:
That sounds fine by me. Feel free to use the fancy 'Commit suggestion' box above.